plugins/woocommerce-admin/client/marketplace/components/product-loader/product-loader.scss

```
@@ -0,0 +1,69 @@
+@import '../../stylesheets/_variables.scss';
+
+.woocommerce-marketplace {
+	&__product-loader-cards {
+		display: grid;
+		background: linear-gradient(to right, $gray-0 40%, $gray-10 60%, $gray-0 80%);
```

I like it when it's even more subtle.

```suggestion
		background: linear-gradient(to right, $gray-0 40%, $gray-5 60%, $gray-0 80%);
```